test/e2e/kubernetes/deployment/deployment.go

```diff
@@ -123,7 +123,7 @@ metadata:
     role: %s
   name: %s
 spec:
-  replicas: 1
+  replicas: 9
```

Comment:
artificially setting replica count high here

Reply:
Is this what gets used to set up the nginx pod count?  I presume our issue was something to do with the azure LB not being aware of what nodes the nginx pods are living on.

Reply:
Correct. These are the LB specs we test against the ingress-nginx deployment (w/ 9 replicas in this test):

https://github.com/Azure/aks-engine/blob/master/test/e2e/kubernetes/workloads/ingress-nginx-ilb.yaml
https://github.com/Azure/aks-engine/blob/master/test/e2e/kubernetes/workloads/ingress-nginx-elb.yaml

As you can see, the LB routes to the pod (and eventually the node the pod is scheduled onto) via the app selector.

What's different about your scenario?

Reply:
can we try this with k8s 1.16?

Reply:
Updated CI config, the next run will test 1.15 and 1.16